test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -702,12 +776,15 @@ UCS_TEST_P(test_uct_cm_sockaddr, many_clients_to_one_server)
     EXPECT_EQ(num_clients, (int)m_server->num_eps());
 
     /* Disconnect */
-    for (i = 0; i < num_clients; ++i) {
-        /* first 2 entities are m_server and m_client */
-        client_test = &m_entities.at(2 + i);
-        ASSERT_TRUE(client_test != m_client);
-
-        cm_disconnect(client_test);
+    /* first 2 entities are m_server and m_client */
+    for (i = 0; i < num_clients; i++) {
+        if (i % 2) {
+            /* Start disconnect from the client side */
+            cm_disconnect(&m_entities.at(2 + i));
```

Comment:
maybe just start i from 2? :)
is `num_clients` value correct? it looks like overrun, the same below

Reply:
num_clients is 100 so i goes from 0 to 99 but the entities for this test start from 2.
so either start i from 2 til 101 and go into m_entites from i or do as wrote now. what's the difference..? i think that starting i from 0 is cleaner.

Reply:
Ok